<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect 4</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
     border: 2px solid gray;
     border-radius: 3px;
 }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackAppView">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageGameConf">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="gbChooseGameType">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="rbNewGame">
               <property name="text">
                <string>Create a new local game</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbNewNetworkGame">
               <property name="text">
                <string>Create a new network game</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbNetworkGame">
               <property name="text">
                <string>Join a network game</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbSetupGame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Setup Game</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <item row="0" column="0">
              <widget class="QStackedWidget" name="stackGameType">
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="currentIndex">
                <number>2</number>
               </property>
               <widget class="QWidget" name="pageNewGameSetup">
                <layout class="QGridLayout" name="gridLayout_6">
                 <item row="4" column="0">
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="2" column="0">
                  <widget class="LocalGameConfView" name="localGameConf" native="true"/>
                 </item>
                 <item row="3" column="0">
                  <widget class="QPushButton" name="btnStart">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Start</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1" rowspan="3">
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="pageNewNetworkGameSetup">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="1" column="0">
                  <widget class="QPushButton" name="createGamePushButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Create game</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="NetworkGameView" name="networkGameConfView" native="true"/>
                 </item>
                 <item row="1" column="2">
                  <widget class="QPushButton" name="btnManageIndexServers">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>Manage Index Servers...</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="1" colspan="2">
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="pageJoinGameSetup">
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="0" column="0">
                  <widget class="JoinGameConfView" name="joinGameConf" native="true"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="gbStatus">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="lblStatus">
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NetworkGameView</class>
   <extends>QWidget</extends>
   <header>networkgameview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LocalGameConfView</class>
   <extends>QWidget</extends>
   <header>localgameconfview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JoinGameConfView</class>
   <extends>QWidget</extends>
   <header>joingameconfview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rbNewGame</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>gameTypeSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNetworkGame</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>gameTypeSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStart</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>720</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnManageIndexServers</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewIndexServersClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>584</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNewNetworkGame</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>gameTypeSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createGamePushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>createNetworkGameClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>583</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>viewIndexServersClicked()</slot>
  <slot>gameTypeSelectionChanged()</slot>
  <slot>startClicked()</slot>
  <slot>joinClicked()</slot>
  <slot>refreshClicked()</slot>
  <slot>createNetworkGameClicked()</slot>
  <slot>localGame3dToggled(bool)</slot>
  <slot>player2Clicked()</slot>
  <slot>player1Clicked()</slot>
 </slots>
</ui>
